File: Java-Source/Day7.java
//DAY 7 OF HACKERRANK 30 DAY CHALLENGE: ARRAYS (JAVA)
//NOTE: The name of the Solution class template was changed to Day7 for organization in the directory.

import java.util.*;

public class Day7 {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int[] arr = new int[n];
        for(int i=0; i < n; i++){
            arr[i] = in.nextInt();
        }
        in.close();

        for (int x = n-1; x >= 0; x--) {
            System.out.print(arr[x] + " ");
        }
        System.out.print("\n");
    }
}

File: CPP-Source/Day7.cpp
//DAY 7 OF HACKERRANK 30 DAY CHALLENGE: ARRAYS (C++)

#include <string>
#include <vector>
#include <iostream>

using namespace std;


int main() {
	int n;
	cin >> n;
	vector<int> arr(n);
	for (int arr_i = 0; arr_i < n; arr_i++) {
		cin >> arr[arr_i];
	}

	for (int x = n - 1; x >= 0; x--) {
		cout << arr[x] << " ";
	}
	cout << "\n";

	return 0;
}
